app/services/facebook_authenticator.rb

```diff
@@ -12,10 +12,14 @@ def initialize(auth_code)
   end
 
   def authenticate!
+    access_token  = get_access_token(@auth_code)
```

Comment:
No need to pass an instance variable around as that's going to be accessible in `get_access_token` anyway.